### PR TITLE
Lmaoo170_10

### PR DESCRIPTION
**Pull request by tufanbutuner**

### CODE REVIEW

File: Script/dateController.js [hunk 1]
```diff
@@ -3,14 +3,36 @@ $(document).ready(function()
...
 function loadDates()
 {
   var ticketId = new URL(window.location.href).searchParams.get("ticketId");
 
   loadDatesFromServer(ticketId)
   .then(response => 
   {
-    $("#createDate").html(response.data[0].created);
-    $("#updateDate").html(response.data[0].updated);
+    $("#createDate").html(tidyUpTimestamp(response.data[0].created.toLocaleString('en-GB')));
```
Comment (OnikNoor97) [inline Script/dateController.js:35]: Apply toLocalString in the tidyUpTimeStamp function

File: Script/dateController.js [hunk 2]
```diff
@@ -3,14 +3,36 @@ $(document).ready(function()
...
+{
+    var dateCreated = new Date(timestamp);
+    dateCreated.toString();
+    var date = dateCreated.getDate();
+    var month = dateCreated.getMonth();
+    var year = dateCreated.getFullYear();
+    var hours = dateCreated.getHours();
+    var minutes = addZero(dateCreated.getMinutes());
+    var ampm = (hours >= 12) ? "PM" : "AM";
+    // var seconds = addZero(dateCreated.getSeconds());
```
Comment (OnikNoor97) [inline Script/dateController.js:21]: Get Rid

File: Script/dateController.js [hunk 3]
```diff
@@ -3,14 +3,36 @@ $(document).ready(function()
...
+
+function tidyUpTimestamp(timestamp) 
+{
+    var dateCreated = new Date(timestamp);
+    dateCreated.toString();
+    var date = dateCreated.getDate();
+    var month = dateCreated.getMonth();
+    var year = dateCreated.getFullYear();
+    var hours = dateCreated.getHours();
+    var minutes = addZero(dateCreated.getMinutes());
```
Comment (OnikNoor97) [inline Script/dateController.js:19]: No need to make a one line function, apply the function in the minute var using ternary operator
var minute = (dateCreated.getMinutues() <  10) ? i = "0" + i : i;

**Review by tufanbutuner — COMMENTED** [review]
Changed

File: Script/dateController.js [hunk 4]
```diff
@@ -3,14 +3,31 @@ $(document).ready(function()
...
+function tidyUpTimestamp(timestamp) 
+{
+    var dateCreated = new Date(timestamp);
+    dateCreated.toString();
+    var date = dateCreated.getDate();
+    var month = dateCreated.getMonth();
+    var year = dateCreated.getFullYear();
+    var hours = dateCreated.getHours();
+    var minutes = dateCreated.getMinutes();
+    (minutes < 10) ? minutes = "0" + minutes : minutes; // to add 0 from 0-9 minutes
```
Comment (OnikNoor97) [inline Script/dateController.js:15]: var minutes = ("0" + dateCreated.getMins()  : dateCreated.getMins();

**Review by OnikNoor97 — CHANGES_REQUESTED** [review]

**Review by OnikNoor97 — APPROVED** [review]

**Pull request merged**